docs/src/index.md

```diff
@@ -31,6 +31,11 @@ using Oceananigans
 using Oceananigans.Units
 using CairoMakie
 using Breeze
+using Random: seed!
+
+# Fix the seed to generate the noise, for reproducible simulations.
+# You can try different seeds to explore different noise patterns.
+seed!(491)
```

Comment:
```suggestion
seed!(42)
```

Reply:
I had generated the seed randomly just not to always pick the usual suspects 😁 
```julia-repl
julia> rand(1:1000)
491
```

Reply:
but its funn